src/content/developers/docs/consensus-mechanisms/pos/keys/index.md

```diff
@@ -0,0 +1,89 @@
+
+This new type of key uses the [Boneh-Lyn-Shacham (BLS) signature scheme](https://wikipedia.org/wiki/BLS_digital_signature). BLS enables a very efficient aggregation of signatures but also allows reverse engineering of aggregated individual validator keys and is ideal for managing actions between validators.
+
+## The two types of validator key {#two-types-of-key}
+
+Before the switch to proof-of-stake, Ethereum users only had a single elliptic-curve-based private key to access their funds. With the introduction of proof-of-stake, users that wished to be solo stakers also required a **validator key** and a **withdrawal key**.
+
+### The validator key {#validator-key}
+
+The validator signing key consists of two elements:
+
+- Validator **private** key
+- Validator **public** key
+
+The purpose of the validator private key is to actively sign on-chain operations such as block proposals and attestations. Therefore these keys have to be held in a hot wallet.
+
+This flexibility has the advantage to move validator signing keys very quickly from one device to another, however, if they have gotten lost or stolen, a thief may be able to **act maliciously** in the following ways:
+
+- Get the validator slashed) by:
+  - Being a proposer and signing two different beacon blocks for the same slot
+  - Being an attester and signing an attestation that "surrounds" another one.
+  - Being an attester and signing two different attestations having the same target.
+- Force a voluntary exit, which stops the validator from staking, and grants access to its ETH balance to the withdrawal key owner.
+
+The **validator public key** is included in the _deposit data_ which allows Ethereum to identify the validator.
+
+### The withdrawal key {#withdrawal-key}
+
+The withdrawal key will be required to move the validator balance after this is enabled in the upcoming Shanghai upgrade. Just like the validator keys, the withdrawal keys also consist of two components:
+
+- Withdrawal **private** key
+- Withdrawal **public** key
+
+Losing this key means losing access to the validator balance. However, the validator can still sign attestations and blocks since these actions require the validator private key, but there is little to no incentive to do so if the keys are lost.
```

Comment:
```suggestion
Losing this key means losing access to the validator balance. However, the validator can still sign attestations and blocks since these actions require the validator's private key, but there is little to no incentive if the keys are lost.
```